### PR TITLE
Typography user show

### PR DESCRIPTION
**Pull request by DavidKang**
We applied some changes regarding the contrast in the username. Apart from that, we grouped the email with the other elements at the bottom.

#### Before
Screenshot_2019-08-20 Home of Admin(1)

#### After
Screenshot_2019-08-23 Home of Admin(2)

**Beta Program Tooltip:**
Screenshot_2019-08-23 Home of Admin(5)

### CODE REVIEW

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 1]
```diff
@@ -2,65 +2,58 @@
...
     = image_tag_for(user)
 
-    %h4.home-usertitle.mt-4
-      #home-realname
-        = user.realname
-    %h4.d-inline-block#home-login
-      = user.login
+    - if is_user && user.in_beta?
+      %p.bg-info.text-light.text-center
+        In public beta program
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/user/_info.html.haml]: I also like the beta program, but it's not *that* important.

**Comment by saraycp** [comment]
@hennevogel @dmarcoux @DavidKang An alternative for the user's info area could be:

Screenshot-2019-8-21 Home of Admin(1)

I simplified the beta stuff using only one switch instead of a link and a badget.

 I added a new commit that can be squashed after review.

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 2]
```diff
@@ -2,65 +2,60 @@
...
-      - if configuration.accounts_editable?(user)
-        - if account_edit_link.present?
-          = link_to account_edit_link do
-            %i.fas.fa-user-edit
-            Edit your account
+      .mt-4
+        - if user.in_beta?
+          = link_to(user_save_path(user: { login: user.login, in_beta: 0 }), method: :post) do
+            %i.fas.fa-toggle-on.text-primary
+            %span Leave public beta program
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/user/_info.html.haml]: You should make sure it's [obvious](https://uxplanet.org/checkbox-vs-toggle-switch-7fc6e83f10b8) what's on and what is off. I would suggest: Enable Beta Program.

**Comment by DavidKang** [comment]
@hennevogel, what about this approach, regarding the "beta program"
Screenshot_2019-08-23 Home of Admin

**Comment by hennevogel** [comment]
@DavidKang awesome

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 3]
```diff
@@ -2,65 +2,64 @@
...
+      .mt-4
+        = form_tag user_save_path, id: 'beta-form' do
+          .custom-control.custom-switch
+            = hidden_field_tag 'user[in_beta]', false
+            = check_box_tag('user[in_beta]', !user.in_beta, user.in_beta, class: 'custom-control-input', id: 'beta-switch')
+            = label_tag 'Public Beta Program', nil, class: 'custom-control-label', for: 'beta-switch'
+            = hidden_field_tag('user[login]', user.login)
+            %i.fa.fa-question-circle.text-info{ data: { placement: 'top', toggle: 'popover', html: 'true',
+                                                      content: 'Enabling <strong>public beta program</strong>, you will be able to try ' + |
+                                                               'the latest features ongoing development and give us feedback on them.' } } |
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/user/_info.html.haml]:
```suggestion
%i.fa.fa-question-circle.text-info{ data: { placement: 'top', toggle: 'popover', html: 'true',
                                                               content: 'As participant in our <strong>beta program</strong>, you can try ' +
                                                               'the latest features we develop so you can give us feedback before we release them.' } } 
```

File: src/api/app/views/webui2/webui/user/_info.html.haml [hunk 4]
```diff
@@ -2,65 +2,64 @@
...
-          %i.fas.fa-rss-square
-          RSS for notifications
+      .mt-4
+        = form_tag user_save_path, id: 'beta-form' do
+          .custom-control.custom-switch
+            = hidden_field_tag 'user[in_beta]', false
+            = check_box_tag('user[in_beta]', !user.in_beta, user.in_beta, class: 'custom-control-input', id: 'beta-switch')
+            = label_tag 'Public Beta Program', nil, class: 'custom-control-label', for: 'beta-switch'
+            = hidden_field_tag('user[login]', user.login)
+            %i.fa.fa-question-circle.text-info{ data: { placement: 'top', toggle: 'popover', html: 'true',
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/user/_info.html.haml]:
```suggestion
            %i.fa.fa-question-circle.text-info{ data: { placement: 'top', toggle: 'popover', html: 'true',
                                                                           content: 'In our <strong>beta program</strong> you can try ' +
                                                                           'the latest features we develop, so you can give us feedback before we release them.' } } 
```

**Pull request merged**